manifests/4.6/logging.openshift.io_clusterlogforwarders_crd.yaml

```diff
@@ -18,13 +18,25 @@ spec:
   - name: v1
     schema:
       openAPIV3Schema:
-        description: "ClusterLogForwarder is an API to configure forwarding logs. \n You configure forwarding by specifying a list of `pipelines`, which forward from a set of named inputs to a set of named outputs. \n There are built-in input names for common log categories, and you can define custom inputs to do additional filtering. \n There is a built-in output name for the default openshift log store, but you can define your own outputs with a URL and other connection information to forward logs to other stores or processors, inside or outside the cluster. \n For more details see the documentation on the API fields."
+        description: "ClusterLogForwarder is an API to configure forwarding logs.
```

Comment:
@vparfonov have you rebased recently?  I would be surprised if you  any of the manifest files get updated

Reply:
Something did happen recently that changed the description indentation in all the manifests, I suspect an update to the operator-sdk tool. @periklis is that plausible?

Reply:
@vparfonov try `make tools` to ensure your versions of the tools are all up to date.

Reply:
I can confirm that the current operator-sdk-v0.18.2 does indent all the description text in the CRDs like @vparfonov is seeing. As long as that's all that changed, you can merge them.

Reply:
fixed